README.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+
+*This is an Opentrons fork of the open-source [Smoothieware](https://github.com/smoothieware/smoothieware)*
+
+[Recent Stable Build Here](https://github.com/Opentrons/SmoothiewareOT/tree/edge/FirmwareBin)
+
+Smoothie is a free, opensource, high performance G-code interpreter and CNC controller written in Object-Oriented C++ for the LPC17xx micro-controller ( ARM Cortex M3 architecture ). It will run on a mBed, a LPCXpresso, a SmoothieBoard, R2C2 or any other LPC17xx-based board. The motion control part is a port of the awesome grbl.
+
+For more information on smoothie as a whole, please check out their documentation [here](http://smoothieware.org/)
+
+### Materials
+Opentrons utilizes the `LPC1768` chip
+
+### Quick Start
+These are the quick steps to get OT Smoothie dependencies installed on your computer:
+* Pull down a clone of the Opentrons Smoothie github fork project to your local machine.
+* Install [GNU-ARM compiler](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) for your Operating System.
+* Ensure that your path is correctly set by searching for the location of: `arm-none-eabi-gcc/../../bin`
```

Comment:
Not sure what that path is?

Reply:
Trying my best to do an arbitrary path to the gcc bin directory 🙃